src/containers/find-offer/offer-container/OfferContainer.tsx

```diff
@@ -14,25 +15,29 @@ const OfferContainer: FC<OfferContainerProps> = ({ viewMode, offerCards }) => {
     console.log(id)
   }
 
-  const arrayOfCards = offerCards.map((el) => (
+  const columnNumber = viewMode === CardsViewEnums.Grid ? 12 : 1
+
+  const cardsArray = offerCards.map((el) => (
```

Comment:
it is not cardsArray, it is offerCards or offerItems

Reply:
Fixed.